.github/workflows/ci.yml

```diff
@@ -38,25 +38,47 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
-      - uses: actions-rs/cargo@v1
+      - uses: actions/cache@v2
         with:
-          command: test
-          args: --release --all-features --manifest-path=compact_str/Cargo.toml -- --include-ignored
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ github.event.repository.name }}-${{ runner.os }}-cargo-test-v3
```

Comment:
I don't think we need the repo name here, but we should include a hash of the `Cargo.lock`, [example](https://github.com/actions/cache/blob/main/examples.md#rust---cargo). This way if the dependencies change it invalidates the cache

Reply:
Done!